nameko/testing/utils.py

```diff
@@ -89,6 +90,13 @@ def assert_stops_raising(fn, exception_type=Exception, timeout=10,
             eventlet.sleep(interval)
 
 
+MockCallArgs = namedtuple("MockCallArgs", ["positional", "keyword"])
+
+
+def unpack_mock_call(call):
```

Comment:
I think it's slightly easier to interpret using the named attribute rather than remembering that it unpacks to `args, kwargs`. If you're only interested in one or the other you don't have to do `args, _ = call` or whatever.

I'm not wedded to it if you'd rather not introduce a new helper though.

Really I just want Mock's `call` to be a namedtuple out of the box.